libraries/AP_HAL_ChibiOS/hwdef/Swan-K1/defaults.parm

```diff
@@ -97,9 +97,9 @@ Q_A_RAT_YAW_SMAX      15
 Q_A_ACCEL_P_MAX    40000
 Q_A_ACCEL_R_MAX    20000
 Q_A_ACCEL_Y_MAX    27000
-Q_P_ACCZ_P            0.8
-Q_P_POSZ_P            1.5
-Q_P_VELZ_P            8.0
+Q_P_ACCD_P            0.08
+Q_P_POSD_P            0.15
```

Comment:
It looks like we're being inconsistent about which PID gains are reduced.  Does it depend upon the vehicle?

For example in the skyviper-journey (above), PSC_POSZ_P becomes PSC_POSD_P but the gain is not changed.  But here in the Swan-K1 quad plane is seems like Q_P_POSZ_P becomes Q_P_POSD_P and is also made 10x smaller